CHANGELOG.md

```diff
@@ -69,7 +79,8 @@ import {autobind} from '@shopify/javascript-utilities/decorators';
 
 ---
 
-[unreleased]: https://github.com/shopify/javascript-utilities/compare/v2.4.1...HEAD
+[unreleased]: https://github.com/shopify/javascript-utilities/compare/v2.4.2...HEAD
+[2.4.2]: https://github.com/shopify/javascript-utilities/compare/v2.3.0...v2.4.2
```

Comment:
I'm unfamiliar with how the change log works for versioning here. The `[unreleased]` line makes sense to me but I'm not sure about the `[2.4.2]`. @ismail-syed should be able to answer this 😄

Reply:
This would be a v2.5.0 change since there are new features from the Added section.